extensions/blocks/podcast-player/components/header.js

```diff
@@ -1,67 +1,95 @@
 /**
  * External dependencies
  */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
 import { memo } from '@wordpress/element';
 
 const Header = memo(
-	( { playerId, title, cover, link, track, children, showCoverArt, showEpisodeDescription } ) => (
-		<div className="jetpack-podcast-player__header-wrapper">
-			<div className="jetpack-podcast-player__header" aria-live="polite">
-				{ showCoverArt && cover ? (
-					<div className="jetpack-podcast-player__track-image-wrapper">
+	( { playerId, title, cover, link, track, children, showCoverArt, showEpisodeDescription, colors } ) => (
+		<div className="jetpack-podcast-player__header">
+			<div className="jetpack-podcast-player__current-track-info">
+				{ showCoverArt && cover && (
+					<div className="jetpack-podcast-player__cover">
 						{ /* alt="" will prevent the src from being announced. Ideally we'd have a cover.alt, but we can't get that from the RSS */ }
-						<img className="jetpack-podcast-player__track-image" src={ cover } alt="" />
+						<img className="jetpack-podcast-player__cover-image" src={ cover } alt="" />
 					</div>
-				) : null }
+				) }
 
-				{ title || ( track && track.title ) ? (
-					<div className="jetpack-podcast-player__titles">
-						<Title playerId={ playerId } title={ title } link={ link } track={ track } />
-					</div>
-				) : null }
+				{ !! ( title || ( track && track.title ) ) && (
+					<Title
+						playerId={ playerId }
+						title={ title }
+						link={ link }
+						track={ track }
```

Comment:
> Is it ever possible to have situations where title or track does not exist? 

I don't think this can happen. I'm not 100% sure what you are suggesting. To always call `<Title>` and let it possibly return null when it doesn't have data it requires?

Reply:
Maybe pass `track.title` to the component when `title` is not available and only `track.title` is?

Reply:
I think there is a confusion here. `title` is the podcast title, while `track` contains the current track title. I'm going to do minor changes here.

Reply:
`Title` handles missing `titles` internally, I'm not sure we need to account for it here. Usually I'd advocate for dumb child components, but in this case it's adding HTML (or not), based on whether there is a title (or not)

Reply:
I think the current implementation is ok in terms of functionality. It means handles properly if either `title` or `track.title` are undefined.
However, I think we should add a `trackTitle` or something like this and avoid passing the whole `track` object to the `<Title />` component.